opencraft/settings.py

```diff
@@ -684,6 +684,9 @@
 AWS_SECRET_ACCESS_KEY = env('AWS_SECRET_ACCESS_KEY', default=None)
 AWS_ACCESS_KEY_ID = env('AWS_ACCESS_KEY_ID', default=None)
 AWS_S3_BUCKET_PREFIX = env('S3_BUCKET_PREFIX', default='ocim')
+AWS_S3_CUSTOM_REGION_HOSTNAME = 's3.{region}.amazonaws.com'
+AWS_S3_DEFAULT_HOSTNAME = 's3.amazonaws.com'
```

Comment:
I thought about this but decided against doing it because the values are not environment-specific (imho) unless we start using S3-compatible backends which work with `boto`.